.github/workflows/create_wappalyzer_update.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         cp -R wappalyzer/src/icons/ zap-extensions/addOns/wappalyzer/src/main/resources/org/zaproxy/zap/extension/wappalyzer/resources/icons
         cp -f wappalyzer/src/apps.json zap-extensions/addOns/wappalyzer/src/main/resources/org/zaproxy/zap/extension/wappalyzer/resources
         cd zap-extensions
+        ./gradlew :addOns:wappalyzer:updateChangelog --change=" - Update with Wappalyzer icon and pattern changes."
```

Comment:
This is what the 05-03 update used, would it be better to say "Updated with upstream Wappalyzer icon and pattern changers." ?

(I can do a Wappalyzer release so that we won't end-up with a 'duplicate' entry next month.)

Reply:
Looks good.

The task will not add duplicates if the change is the same, if not worth releasing the add-on we can tweak the changelog accordingly.

Reply:
I meant if I change it to slightly better text without doing a release then the new message would be added next month even though it means basically the same thing...

Reply:
I decided to use the revised text :wink:, if a release isn't done I can always remove the less specific change entry manually.

Reply:
Or tweak it to match the new, then it will not be duplicated.